internal/xpkg/fetch.go

```diff
@@ -135,7 +135,7 @@ func WithServiceAccount(sa string) FetcherOpt {
 func NewK8sFetcher(client kubernetes.Interface, opts ...FetcherOpt) (*K8sFetcher, error) {
 	k := &K8sFetcher{
 		client:    client,
-		transport: remote.DefaultTransport.Clone(),
+		transport: remote.DefaultTransport.(*http.Transport).Clone(),
```

Comment:
Did the type of the default transport change from a concrete type to an interface?

Reply:
Looks like they switched it to be a Roundtripper instead, https://github.com/google/go-containerregistry/commit/e3b94c7e7a5720c5e2985012f617f36006abff62#